### ISSUE TITLE
Wordiness of Both Guides

### ISSUE THREAD

**Issue by abhityakrishnaraj** [abhityakrishnaraj/pe#10]
I would say that both the user guide and the developer guide are a bit wordy and sometimes repeats information that has already been established in later sections of the guide. I think defining all of the variances of entering the variables in a separate section and then utilizing those variables in the definition of the commands could really cut down on the overall visual clutter.

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Description provided is vague. Developer guide is meant to be comprehensive. For the user guide, we intend command sections to be standalone to minimize the amount of jumping around that the user needs to do. To spread out the information density, we also provide a quick start at the start and a command summary at the end so that the user can take in information slowly.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

-------------------
